### ISSUE TITLE
App still maintained?

### ISSUE THREAD

**Issue by IzzySoft** [kounkou/Staples#47]
Do you still maintain this app, @kounkou? There was no commit for almost 2 years, so I wonder if you stopped this project.

**Comment by kounkou** [comment]
Hi @IzzySoft,
Thanks for asking.
I would say paused instead, I found new ways to manage the REST part,
So will probably push things soon.
Feel free to contribute if you have any ideas in mind.
I will close the issue if that answers your question.
Best

**Comment by IzzySoft** [comment]
Yes it does answer my question, @kounkou – thanks a lot! And not being a developer, I cannot contribute to the code. My contribution [is a little different](https://apt.izzysoft.de/fdroid/index/apk/org.qtproject.example), but I hope you like it :smiley:

**Comment by kounkou** [comment]
That’s amazing ! 😆 🙏🏾

**Comment by IzzySoft** [comment]
Glad you like it! If you create a new tag and add the APK to it, my updater should grab that automatically within 24h – and it will be available for users via the F-Droid app (if they have my repo enabled). They will even be automatically informed about updates if they already have the app installed.

**Comment by kounkou** [comment]
That's nice. Is there anyway to see the number of downloads ?
If possible, can you show me how to add an application myself ?
iFresh (https://github.com/kounkou/iFresh) is the app you wanna put on the package manager.

**Comment by IzzySoft** [comment]
> Is there anyway to see the number of downloads ?

Nope. Neither that, nor the number of users.

> If possible, can you show me how to add an application myself ?

https://gitlab.com/IzzyOnDroid/repo (the README has the details)

> Fresh (kounkou/iFresh) is the app you wanna put on the package manager.

Looks as if it would fit – but no commit for almost 2 years raises the same question as here: Is it still maintained?

**Comment by kounkou** [comment]
iFresh has a stable version. I performed some tests a week ago to ensure the app is still working.
other than that, I didn't see any additions for the app for now.
Thanks for the link.

**Comment by IzzySoft** [comment]
Oh, I too have projects I didn't touch for quite a while (though I archived most of them), and other projects I touch only occasionally (but still maintain them in a way – that is, when I see something missing/wrong I'll add/fix it, reply to issues opened, process PRs etc). That's what I'd answer on the question whether such a project is still maintained. As you wrote: the question is justified – whether the answer is yes or no :wink:

TL;DR: It's you to say whether it's still maintained or not, and especially if it still works (sometimes there are API issues – concerning the Android API as well as those APIs used to gather data from other services).
